schema/properties_directory/workflow/convergence/ionic.json

```diff
@@ -6,7 +6,8 @@
         "name": {
             "enum": [
                 "convergence_ionic"
-            ]
+            ],
+            "$comment": "isGenerative:true"
```

Comment:
I think we should explain in `README.md` what `isGenerative` means and is used for.

Reply:
section added to `README.md` explaining purpose of flag and linking to example files